proto/cluster.proto

```diff
@@ -211,6 +211,12 @@ message HeadGroupSpec {
   map<string, string> ray_start_params = 4;
   // Optional. The volumes mount to head pod
   repeated Volume volumes = 5;
+  // Optional. Environment variables for head pod
+  map<string, string> environment = 6;
```

Comment:
@blublinsky in the L166 there is a envs for the whole cluster, so here I would like to double comfirm that is this used for specific config of head and worker, in this situation head and worker may have some different envs/label/annotation?

Reply:
Yes, they are specified independently for head and different worker groups